config/webpack/environment.js

```diff
@@ -1,3 +1,11 @@
-const { environment } = require('@rails/webpacker')
+const {environment} = require('@rails/webpacker')
 
+const webpack = require('webpack')
+environment.plugins.append("Provide",
```

Comment:
Đổi nháy kép thành nháy đơn ông nhé

Reply:
Như tôi nói thì tôi chưa thấy công ty bắt convention, và JS chuẩn cũng không khắt khe về cái vấn đề này nên tôi bỏ qua nhá :v Vì trước nay làm FE cũng toàn dùng "" cho nó đồng bộ